### PR TITLE
Added Grafana Play Links to Panel visualization docs

### PR DESCRIPTION
**Pull request by srperf**
**What is this feature?**

Added Grafana Play links and messages to panel visualizations docs (Alert, News, Node, Traces) that did not have them yet.

**Why do we need this feature?**

Users can now check examples about the documentation they are reading.

**Who is this feature for?**

Everyone

**Which issue(s) does this PR fix?**:

None

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by usmangt — APPROVED** [review]
LGTM !!

File: docs/sources/panels-visualizations/visualizations/traces/index.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ For more information about traces and how to use them, refer to the following do
 
 {{< figure src="/static/img/docs/explore/trace-view-9-4.png" class="docs-image--no-shadow" max-width= "900px" caption="Screenshot of the trace view" >}}
 
+{{< docs/play title="Grafana Traces Visualization" url="https://play.grafana.org/d/edodkfmj0tce8f/" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/traces/index.md:68]:
```suggestion
{{< docs/play title="Traces Panel" url="https://play.grafana.org/d/edodkfmj0tce8f/" >}}
```

File: docs/sources/panels-visualizations/visualizations/node-graph/index.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The following video provides beginner steps for creating node panel visualizatio
 
 {{< youtube id="VrvsMkRwoKw" >}}
 
+{{< docs/play title="Node Panel Visualizations in Grafana" url="https://play.grafana.org/d/bdodfbi3d57uoe/" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/node-graph/index.md:32]:
```suggestion
{{< docs/play title="Node graph panel" url="https://play.grafana.org/d/bdodfbi3d57uoe/" >}}
```

File: docs/sources/panels-visualizations/visualizations/news/index.md [hunk 3]
```diff
@@ -30,6 +30,8 @@ In version 8.5, we discontinued the "Use Proxy" option for Grafana news visualiz
 
 You can use the news visualization to provide regular news and updates to your users.
 
+{{< docs/play title="Grafana News Visualization" url="https://play.grafana.org/d/cdodkwspaaa68b/" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/news/index.md:33]:
```suggestion
{{< docs/play title="News Panel" url="https://play.grafana.org/d/cdodkwspaaa68b/" >}}
```

File: docs/sources/panels-visualizations/visualizations/alert-list/index.md [hunk 4]
```diff
@@ -28,12 +28,14 @@ refs:
...
 # Alert list
 
-Alert lists allow you to display a list of important alerts that you want to track. You can configure the alert list to show the current state of your alert, such as firing, pending, or normal. Learn more about alerts in [Grafana Alerting overview][].
+Alert lists allow you to display a list of important alerts that you want to track. You can configure the alert list to show the current state of your alert, such as firing, pending, or normal. Learn more about alerts in [Grafana Alerting overview](https://grafana.com/docs/grafana/latest/alerting/).
 
 {{< figure src="/static/img/docs/alert-list-panel/alert-list-panel.png" max-width="850px" alt="An alert list visualization" >}}
 
 On each dashboard load, this visualization queries the alert list, always providing the most up-to-date results.
 
+{{< docs/play title="Grafana Alert List Visualization" url="https://play.grafana.org/d/bdodlcyou483ke/" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/alert-list/index.md:37]:
```suggestion
{{< docs/play title="Alert List" url="https://play.grafana.org/d/bdodlcyou483ke/" >}}
```

**Review by imatwawana — APPROVED** [review]
Thanks for the additions! I've updated the Play link text so that it matches the name of the Play dashboard exactly per the style guide and changed the style of the link you updated. Otherwise, this looks good!

**Pull request merged**